Lint warning

### Node version (or tell us if you're using electron or some other framework):
6.9.1
### ShellJS version (the most recent version/Github branch you see the bug on):
0.7.5
### Operating system:
OS X 10.11.6
### Description of the bug:
There's a lint warning on line 50 of `src/rm.js`
### Example ShellJS command to reproduce the error:

```
$ npm run lint

> shelljs@0.7.5 lint /Users/freitagb/src/javascript/shelljs
> eslint .

/Users/freitagb/shelljs/src/rm.js
  50:5  warning  Unexpected constant condition  no-constant-condition

✖ 1 problem (0 errors, 1 warning)
```

--- a/src/rm.js
+++ b/src/rm.js
@@ -47,6 +47,8 @@ function rmdirSyncRecursive(dir, force) {
   try {
     // Retry on windows, sometimes it takes a little time before all the files in the directory are gone
     var start = Date.now();
+
+    // TODO: replace this with a finite loop
     for (;;) {
       try {
         result = fs.rmdirSync(dir);

--- a/src/rm.js
+++ b/src/rm.js
@@ -47,7 +47,7 @@ function rmdirSyncRecursive(dir, force) {
   try {
     // Retry on windows, sometimes it takes a little time before all the files in the directory are gone
     var start = Date.now();
-    while (true) {
+    for (;;) {
       try {
         result = fs.rmdirSync(dir);
         if (fs.existsSync(dir)) throw { code: 'EAGAIN' };

